cves/CVE-2003-0083.yml

```diff
@@ -126,10 +127,10 @@ interesting_commits:
     interesting in light of the lessons learned from this vulnerability. Any
     emerging themes?
   commits:
-    - commit:
-      note:
-    - commit:
-      note:
+    - commit: 5430f8800f5fffd57e7421dee0ac9de8ca4f9573
+      note: This began the baseline for a move to apache 2.0. This is a major 
+            event in the evolution of the project and could of been a very busy
```

Comment:
Instead of "could of", consider using "could have" :)

Reply:
Sounds good, will make change